app/assets/javascripts/surveys.js

```diff
@@ -110,6 +110,123 @@
+            '^/government/news/defence-secretary-announces-five-year-plan-for-key-military-sites',
+            '^/government/news/proposed-new-timetable-for-state-pension-age-increases',
+            '^/government/news/uk-statement-on-re-election-of-president-buhari-in-nigeria',
+            '^/government/organisations/hm-revenue-customs/contact/child-benefit',
+            '^/government/organisations/hm-revenue-customs/contact/vat-online-services-helpdesk ',
+            '^/government/organisations/home-office',
+            '^/government/organisations/medicines-and-healthcare-products-regulatory-agency',
+            '^/government/organisations/public-health-england',
+            '^/government/publications/advisory-fuel-rates',
+            '^/government/publications/advisory-fuel-rates/advisory-fuel-rates-from-1-march-2016',
+            '^/government/publications/general-information-about-checks-by-compliance-centres-ccfs1b ',
+            '^/government/publications/guidance-on-applying-for-uk-visa-approved-english-language-tests',
+            '^/government/publications/introducing-govuk-verify/introducing-govuk-verify',
+            '^/government/publications/key-stage-2-tests-2018-mathematics-test-materials',
+            '^/government/publications/statutory-sick-pay-employees-statement-of-sickness-sc2',
+            '^/guidance/rates-and-thresholds-for-employers-2019-to-2020',
+            '^/guidance/the-highway-code',
+            '^/hand-luggage-restrictions ',
+            '^/holiday-entitlement-rights ',
+            '^/housing-and-universal-credit',
+            '^/inheritance-tax',
+            '^/inheritance-tax/gifts',
+            '^/limited-company-formation/register-your-company',
+            '^/make-court-claim-for-money/court-fees',
+            '^/marriage-allowance/how-to-apply',
+            '^/married-couples-allowance',
+            '^/maternity-pay-leave/leave',
+            '^/national-insurance/how-much-you-pay',
+            '^/news-and-communications',
+            '^/pay-dartford-crossing-charge/charges-fines',
+            '^/pay-self-assessment-tax-bill/bank-details',
+            '^/pay-vat',
+            '^/power-of-attorney',
+            '^/register-employer ',
+            '^/register-offices',
+            '^/research-family-history',
+            '^/schools-admissions/admissions-criteria',
+            '^/scrapped-and-written-off-vehicles ',
+            '^/search/advanced',
+            '^/self-assessment-forms-and-helpsheets',
+            '^/self-assessment-tax-returns/sending-return',
+            '^/settled-status-eu-citizens-families/after-youve-applied',
+            '^/stamp-duty-land-tax',
+            '^/stamp-duty-land-tax/residential-property-rates',
+            '^/statutory-sick-pay/what-youll-get',
+            '^/tax-on-dividends',
+            '^/tell-hmrc-change-address',
+            '^/theory-test',
+            '^/theory-test/revision-and-practice ',
```

Comment:
I don't think it matters, but this line and a few others have an empty space at the end which might be worth removing.

Reply:
Thank you, will remove them